subprojects/common/elastic/src/main/kotlin/com/avito/android/elastic/ElasticServiceFactory.kt

```diff
@@ -6,13 +6,14 @@ import okhttp3.OkHttpClient
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 import retrofit2.create
+import java.net.URL
```

Comment:
why not `HttpUrl`?

Reply:
Agree, this module already knows about okhttp. Will change

Reply:
Can't do it. HttpUrl is non Serializable.